lib/dependencies/CommonJsExportsParserPlugin.js

```diff
@@ -16,11 +16,21 @@ const DynamicExports = require("./DynamicExports");
 const HarmonyExports = require("./HarmonyExports");
 const ModuleDecoratorDependency = require("./ModuleDecoratorDependency");
 
+/** @typedef {import("estree").Super} Super */
+/** @typedef {import("estree").CallExpression} CallExpression */
 /** @typedef {import("estree").Expression} ExpressionNode */
+/** @typedef {import("estree").Expression} Expression */
+/** @typedef {import("estree").AssignmentExpression} AssignmentExpression */
+
 /** @typedef {import("../NormalModule")} NormalModule */
 /** @typedef {import("../javascript/BasicEvaluatedExpression")} BasicEvaluatedExpression */
 /** @typedef {import("../javascript/JavascriptParser")} JavascriptParser */
+/** @typedef {import("./CommonJsDependencyHelpers").CommonJSDependencyBaseKeywords} CommonJSDependencyBaseKeywords */
 
+/**
+ * @param {TODO} expr expression
```

Comment:
Add a TypeGuard here. Going to refactor this today.

Reply:
I attempted to type guard this however I still ran into some struggles. I opted instead to well-document the functions purpose and leave the TODO. We can revisit this, or someone else can try and refactor this code to be easier to type.